src/utils/defaultArrowRenderer.js

```diff
@@ -4,9 +4,12 @@ import PropTypes from 'prop-types';
 export default function arrowRenderer ({ onMouseDown }) {
 	return (
 		<span
-			className="Select-arrow"
 			onMouseDown={onMouseDown}
-		/>
+		>
+			<svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet">
```

Comment:
That arrow will need to be centered
![screen shot 2018-11-26 at 10 46 42 am](https://user-images.githubusercontent.com/1847316/49028939-233f2100-f169-11e8-93c2-25ec985bf7a4.png)

Reply:
do you mean vertically centered?

Reply:
yes